scrapy/pipelines/images.py

```diff
@@ -46,6 +46,7 @@ class ImagesPipeline(FilesPipeline):
     THUMBS = {}
     DEFAULT_IMAGES_URLS_FIELD = 'image_urls'
     DEFAULT_IMAGES_RESULT_FIELD = 'images'
+    DEFAULT_IMAGES_CONVERT_TYPE = 'JPEG'
```

Comment:
I would suggest to not introduce another upper-case class attribute, but add a new `IMAGES_CONVERT_TYPE` setting in `default_settings.py`